### PR TITLE
WIP - Move to .NET Core 2.1

### PR DESCRIPTION
**Pull request by vcsjones**

### CODE REVIEW

File: AzureSignTool/SipExtensionFactory.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ AuthenticodeSignerAttributes attributes
             var extension = Path.GetExtension(filePath).ToLower();
             switch (extension)
             {
+                case ".msix":
```
Comment (onovotny) [inline AzureSignTool/SipExtensionFactory.cs]: Do we know if this is really all it is … is it the same SIP? I have to imaging it is?

If so, there's also `.msixbundle` Not sure about the encrypted stuff (if that even works...)

File: AzureSignTool/AzureSignTool.csproj [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
     <Authors>Kevin Jones</Authors>
     <Description>Azure SignTool</Description>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>AzureSignTool.snk</AssemblyOriginatorKeyFile>
+    <RuntimeIdentifiers>win10-x64;win10-x86</RuntimeIdentifiers>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="3.17.2" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
```
Comment (onovotny) [inline AzureSignTool/AzureSignTool.csproj]: There's a 3.0 version of this library now

File: AzureSignTool/SipExtensionFactory.cs (hunk 1 above)
Comment (vcsjones) [reply AzureSignTool/SipExtensionFactory.cs]: Bah that change wasn't support to land.

**Comment by vcsjones** [comment]
Closing in favor of #33.

**Comment by onovotny** [comment]
@vcsjones it'd still be nice if could fail to install on non-windows instead of being a runtime error

**Comment by vcsjones** [comment]
@onovotny 

From #31.

>There isn't a way to make dotnet tool install fail on Windows, but it should be pretty straightforward to add a runtime check if someone tries to run the tool on anything else.

Sounds like a feature request.

**Comment by onovotny** [comment]
Adding @kathleendollard. Given the structure of the tools folder, do we have to use `any`? Can we use an RID for the tool to specify the required OS?

**Pull request closed without merging**